resources/definitions/ankermake_m5c.def.json

```diff
@@ -0,0 +1,400 @@
+    "bridge_wall_coast": { "value": 100 },
+    "bridge_wall_speed": { "value": 20 },
+    "bridge_skin_speed": { "value": 20 },
+    "bridge_skin_material_flow": { "value": 100 },
+    "bridge_skin_density": { "value": 110 },
+    "bridge_fan_speed": { "value": 100 },
+    "bridge_enable_more_layers": { "value": true },
+    "bridge_skin_speed_2": { "value": 50 },
+    "bridge_skin_material_flow_2": { "value": 80 },
+    "bridge_skin_density_2": { "value": 80 },
+    "bridge_fan_speed_2": { "value": 100 },
+    "bridge_skin_speed_3": { "value": 50 },
+    "bridge_skin_material_flow_3": { "value": 110 },
+    "bridge_skin_density_3": { "value": 80 },
+    "bridge_fan_speed_3": { "value": 100 },
+    "bridge_wall_material_flow": { "value": 100 },
+    "bridge_split_min_length": { "value": 5 },
+    "bridge_shrink_length": { "value": 0.01 },
+
+    "speed_travel": {
+      "minimum_value": "0.1",
+      "maximum_value_warning": "500",
+      "value": 500
+    },
+    "speed_print": {
+      "minimum_value": "0.1",
+      "maximum_value_warning": "500",
+      "value": 500
+    },
+    "speed_infill": {
+      "minimum_value": "0.1",
+      "maximum_value_warning": "500",
+      "value": 270
+    },
+    "speed_wall": {
+      "minimum_value": "0.1",
+      "maximum_value_warning": "500",
+      "value": 250
+    },
+    "speed_wall_0": {
+      "minimum_value": "0.1",
+      "maximum_value_warning": "500",
+      "value": 150
+    },
+    "speed_wall_x": {
+      "minimum_value": "0.1",
+      "maximum_value_warning": "500",
+      "value": 250
+    },
+    "speed_topbottom": {
+      "minimum_value": "0.1",
+      "maximum_value_warning": "500",
+      "value": 150
+    },
+    "speed_roofing": {
+      "minimum_value": "0.1",
+      "maximum_value_warning": "500",
+      "value": 150
+    },
+    "speed_support": {
+      "minimum_value": "0.1",
+      "maximum_value_warning": "500",
+      "value": 250
+    },
+    "speed_support_infill": {
+      "minimum_value": "0.1",
+      "maximum_value_warning": "500",
+      "value": 250
+    },
+    "speed_wall_x_roofing": {
+      "minimum_value": "0.1",
+      "maximum_value_warning": "500",
+      "value": "speed_wall_x"
+    },
+    "speed_support_interface": {
+      "minimum_value": "0.1",
+      "maximum_value_warning": "500",
+      "value": 166.667
+    },
+    "speed_support_roof": {
+      "minimum_value": "0.1",
+      "maximum_value_warning": "500",
+      "value": 166.667
+    },
+    "speed_support_bottom": {
+      "minimum_value": "0.1",
+      "maximum_value_warning": "500",
+      "value": 166.667
+    },
+    "speed_prime_tower": {
+      "minimum_value": "0.1",
+      "maximum_value_warning": "500",
+      "value": 500
+    },
+    "speed_layer_0": {
+      "minimum_value": "0.1",
+      "maximum_value_warning": "500",
+      "value": 50
+    },
+    "speed_print_layer_0": {
+      "minimum_value": "0.1",
+      "maximum_value_warning": "500",
+      "value": 50
```

Comment:
Why override this value? By default we set this value to inherit the value "speed_layer_0", which is what you seem to want to do here.